recipes/pycalphad/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.4.1" %}
+{% set sha256 = "2603069927f4e6f69038f90a993f810c63585b6bf428473bcd5254d940a449f" %}
```

Comment:
Appears the checksum doesn't match. 😕

Assuming it is not provided for us, this can be computed locally using `openssl sha256 <tarball>` where `<tarball>` is what we are downloading.